workflows/process/metaphinder_own_DB/phage_references_blastDB.nf

```diff
@@ -0,0 +1,31 @@
+process phage_references_blastDB {
+    if (params.cloudProcess) { publishDir "${params.databases}/blast_DB_phage", mode: 'copy', pattern: "blast_database.tar.gz" }
```

Comment:
storeDir. please check all the outputs

Reply:
yes later...
prio was get it to work properly again so people could do the forking